tests/CMakeLists.txt

```diff
@@ -34,7 +35,7 @@ add_executable(jwt-cpp-test ${TEST_SOURCES})
 
 # NOTE: Don't use space inside a generator expression here, because the function prematurely breaks the expression into
 # multiple lines. https://cmake.org/pipermail/cmake/2018-April/067422.html
-set(JWT_TESTER_GCC_FLAGS -Wall -Wextra -Wpedantic)
+set(JWT_TESTER_GCC_FLAGS -Wall -Wextra -Wpedantic -ggdb)
```

Comment:
🤔 https://stackoverflow.com/a/14811348/8480874

Since this is the tester, It makes sense but I am a fan of less is more

Reply:
Shouldn't cmake add flags to generate debug symbols by itself if you build in the debug configuration ?

Reply:
It does `-g0` which is more portable, this is a compiler specific one that has extras